robocop/checkers/naming.py

```diff
@@ -129,6 +135,12 @@ class KeywordNamingChecker(VisitorChecker):
         'else',
         'else if'
     }
+    bdd = {
+        'given',
+        'when',
+        'and',
+        'then'
```

Comment:
One more keyword is also possible: `but`. See [here](http://robotframework.org/robotframework/latest/RobotFrameworkUserGuide.html#behavior-driven-style)

Reply:
Oh good eye, I will update it